contracts/test/v0.8/automation/helpers.ts

```diff
@@ -109,6 +109,11 @@ export const assertSatisfiesInterface = (
       continue
     }
 
+    // addFunds is a payable function starting from v2.3, skipping the stateMutability check
+    if (functionName === 'addFunds(uint256,uint96)') {
```

Comment:
this is my proposal for skipping the payable check for addFunds.

Reply:
works for me for now 👍